CHANGES

```diff
@@ -23,6 +23,7 @@ Features added
 * Show warnings if no domains match with `primary_domain` (ref: #2001)
 * C++, show warnings when the kind of role is misleading for the kind
   of target it refers to (e.g., using the `class` role for a function).
+* public names for latex environments and parameters used by note, warning, and other admonition types, allowing full customizability from either ``'preamble'`` key or ``raw`` directive (ref: feature request #2674)
```

Comment:
using emacs whitespace-mode with whitespace-line-column at 95 I do identify some longer lines, but not earlier than 1.4. Many are my fault, but some not and I have found some lines at 104 chars.